src/bridge/RippleJSBridge.js

```diff
@@ -463,10 +463,29 @@ const RippleJSBridge: WalletBridge<Transaction> = {
 
   getTransactionAmount: (a, t) => t.amount,
 
-  editTransactionRecipient: (account, t, recipient) => ({
-    ...t,
-    recipient,
-  }),
+  editTransactionRecipient: (account, t, recipient) => {
+    const params = new URLSearchParams(recipient)
+    recipient = recipient.split('&')[0]
```

Comment:
my fault in the #1444 but it's actually a split on `?`

e.g.

`rwfGzgd4bUStS9gA5xUhCmg1J86TMtmGMo?dt=51489346`

Reply:
Oh right, that makes more sense I blindly tested using your string, I'll update the PR with those